packages/http/src/resources/__tests__/Hotspots.spec.ts

```diff
@@ -172,6 +174,67 @@ describe('get', () => {
   })
 })
 
+describe('elected', () => {
+  nock('https://api.helium.io')
+    .get('/v1/elected/123456')
+    .reply(200, {
+      data: [
+        hotspotFixture({ name: 'previous-consensus-hotspot-1' }),
+        hotspotFixture({ name: 'previous-consensus-hotspot-2' }),
+        hotspotFixture({ name: 'previous-consensus-hotspot-3' }),
+        hotspotFixture({ name: 'previous-consensus-hotspot-4' }),
+        hotspotFixture({ name: 'previous-consensus-hotspot-5' }),
+        hotspotFixture({ name: 'previous-consensus-hotspot-6' }),
+        hotspotFixture({ name: 'previous-consensus-hotspot-7' }),
+        hotspotFixture({ name: 'previous-consensus-hotspot-8' }),
+        hotspotFixture({ name: 'previous-consensus-hotspot-9' }),
+        hotspotFixture({ name: 'previous-consensus-hotspot-10' }),
+        hotspotFixture({ name: 'previous-consensus-hotspot-11' }),
+        hotspotFixture({ name: 'previous-consensus-hotspot-12' }),
+        hotspotFixture({ name: 'previous-consensus-hotspot-13' }),
+        hotspotFixture({ name: 'previous-consensus-hotspot-14' }),
+        hotspotFixture({ name: 'previous-consensus-hotspot-15' }),
+        hotspotFixture({ name: 'previous-consensus-hotspot-16' }),
+      ],
+    })
+  nock('https://api.helium.io')
+    .get('/v1/elected')
+    .reply(200, {
+      data: [
+        hotspotFixture({ name: 'current-consensus-hotspot-1' }),
+        hotspotFixture({ name: 'current-consensus-hotspot-2' }),
+        hotspotFixture({ name: 'current-consensus-hotspot-3' }),
+        hotspotFixture({ name: 'current-consensus-hotspot-4' }),
+        hotspotFixture({ name: 'current-consensus-hotspot-5' }),
+        hotspotFixture({ name: 'current-consensus-hotspot-6' }),
+        hotspotFixture({ name: 'current-consensus-hotspot-7' }),
+        hotspotFixture({ name: 'current-consensus-hotspot-8' }),
+        hotspotFixture({ name: 'current-consensus-hotspot-9' }),
+        hotspotFixture({ name: 'current-consensus-hotspot-10' }),
+        hotspotFixture({ name: 'current-consensus-hotspot-11' }),
+        hotspotFixture({ name: 'current-consensus-hotspot-12' }),
+        hotspotFixture({ name: 'current-consensus-hotspot-13' }),
+        hotspotFixture({ name: 'current-consensus-hotspot-14' }),
+        hotspotFixture({ name: 'current-consensus-hotspot-15' }),
+        hotspotFixture({ name: 'current-consensus-hotspot-16' }),
+      ],
+    })
+
+  it('retrieves elected hotspots at a given block height', async () => {
+    const client = new Client()
+    const elected = await client.hotspots.elected(123456)
+    expect(elected[0].name).toBe('previous-consensus-hotspot-1')
```

Comment:
might also want to test that for example `.take` works after adding ResourceList above